libs/pipeline.py

```diff
@@ -8,26 +8,49 @@
 
 from dataclasses import dataclass
 import us
+from typing_extensions import final
 
 
+def fips_to_location_id(fips: str) -> str:
+    """Converts a FIPS code to a locationID"""
+    if len(fips) == 2:
+        state_obj = us.states.lookup(fips, field="fips")
+        return f"iso1:us#iso2:us-{state_obj.abbr}"
+    elif len(fips) == 5:
+        state_obj = us.states.lookup(fips[0:2], field="fips")
+        return f"iso1:us#iso2:us-{state_obj.abbr}#fips:{fips}"
+    else:
+        raise ValueError("Bad fips")
+
+
+def cbsa_to_location_id(cbsa_code: str) -> str:
```

Comment:
Can you add a docstring to this, just giving a bit more context about the cbsa code

Reply:
Done.